app/components/user-skills-input.js

```diff
@@ -1,16 +1,28 @@
+import { EKMixin, keyDown, getCode } from 'ember-keyboard';
 import Ember from 'ember';
 
 const {
   Component,
   computed,
-  computed: { alias, and, equal, not, notEmpty },
+  computed: { alias, and, not, notEmpty },
   inject: { service },
   isEmpty,
   run: { once },
-  observer
+  observer,
+  on,
+  TextField
 } = Ember;
 
-export default Component.extend({
+Ember.TextField.extend(EKMixin, {
```

Comment:
@appleJax You destructure `Textfield` above from `Ember` but then explicitly call `Ember.Textfield` here. If you delete `Ember` here, your ESLinting error should pass.